File: Java/Tests/jdi-uitest-tutorialtests/src/test/java/com/epam/jdi/uitests/testing/career/common/tests/RequestInfoPageTests.java
package com.epam.jdi.uitests.testing.career.common.tests;

import com.epam.jdi.uitests.testing.career.page_objects.dataProviders.AttendeeProvider;
import com.epam.jdi.uitests.testing.career.page_objects.dataProviders.CustomerProvider;
import com.epam.jdi.uitests.testing.career.page_objects.entities.Attendee;
import com.epam.jdi.uitests.testing.career.page_objects.entities.Customer;
import com.epam.jdi.uitests.testing.career.page_objects.enums.Countries;
import com.epam.web.matcher.testng.Check;
import org.testng.annotations.BeforeMethod;
import org.testng.annotations.Test;

import java.io.IOException;
import java.lang.reflect.Method;
import java.util.List;

import static com.epam.jdi.uitests.core.preconditions.PreconditionsState.isInState;
import static com.epam.jdi.uitests.testing.career.page_objects.enums.HeaderMenu.CAREERS;
import static com.epam.jdi.uitests.testing.career.page_objects.enums.Preconditions.HOME_PAGE;
import static com.epam.jdi.uitests.testing.career.page_objects.site.EpamSite.*;
import static org.testng.Assert.assertEquals;
/**
 * Created by Iuliia_Petrova on 4/12/2016.
 */
public class RequestInfoPageTests extends TestsBase{

    @BeforeMethod
    public void before(Method method) throws IOException {
        isInState(HOME_PAGE, method);
    }

    @Test(dataProvider = "customers", dataProviderClass = CustomerProvider.class)
    public void sendFeedbackTest(Customer customer) {
        requestInfoPage.open();
        requestInfoPage.checkOpened();
        requestInfoPage.registrationForm.submit(customer);
        requestInfoPage.registrationForm.verify(customer);
    }

}

File: C# .Net/JDI UI Framework/Tests/Scenarios/Tests/RequestInfoPageTests.cs
using System.Runtime.Remoting;
using Epam.Tests.Scenarios.Entities;
using Epam.Tests.Scenarios.Page_Objects;
using Epam.Tests.Scenarios.Page_Objects.Pages;
using Epam.Tests.Scenarios.Page_Objects.Sections;
using NUnit.Framework;
using NUnit.Framework.Constraints;
using static Epam.Tests.Scenarios.Enums.HeaderMenuValues;
using static Epam.Tests.Scenarios.Page_Objects.EpamSite;
using static Epam.JDI.Web.Selenium.DriverFactory.WebDriverUtils;

namespace Epam.Tests.Scenarios.Tests

{
    [TestFixture]
    public class RequestInfoPageTests : TestsBase

{
    [Test]
    public void SendFeedbackTest()
    {
        var customer = new Customer();
        EpamSite.RequestInfoPage.Open();
        EpamSite.RequestInfoPage.CheckOpened();
        EpamSite.RequestInfoPage.RegistrationForm.Submit(customer);
        EpamSite.RequestInfoPage.RegistrationForm.Verify(customer);
    }

}
}